src/edu/washington/escience/myria/operator/agg/CountAllAggregator.java

```diff
@@ -53,4 +56,9 @@ public Object getInitialState() {
     /** The number of tuples seen so far. */
     private long count = 0;
   }
+
+  @Override
+  public void add(final List<TupleBatch> from, final Object state) throws DbException {
+    throw new DbException(" method not implemented");
```

Comment:
I'm not sure but should this be the total number of tuples in `from`? Similar to other places with "method not implemented".